libraries/AP_Networking/AP_Networking_Config.h

```diff
@@ -0,0 +1,84 @@
+#include <AP_HAL/AP_HAL_Boards.h>
+
+#ifndef AP_NETWORKING_ENABLED
+#define AP_NETWORKING_ENABLED 0
+#endif
+
+#if CONFIG_HAL_BOARD == HAL_BOARD_CHIBIOS
+    #define AP_NETWORKING_DHCP_AVAILABLE    LWIP_DHCP
+#else
+    #define AP_NETWORKING_DHCP_AVAILABLE    1 // for non-ChibiOS, assume it's available
+#endif
+
+
+// ---------------------------
+// Below are default params
+// ---------------------------
+
+#ifndef AP_NETWORKING_DEFAULT_OPTIONS
+    #define AP_NETWORKING_DEFAULT_OPTIONS 0
+#endif
+
+// Default DHCP
+#ifndef AP_NETWORKING_DEFAULT_DHCP_ENABLE
+#define AP_NETWORKING_DEFAULT_DHCP_ENABLE 0
+#endif
+
+// Default Static IP Address: 192.168.13.13
+#ifndef AP_NETWORKING_DEFAULT_STATIC_IP_ADDR0
+    #define AP_NETWORKING_DEFAULT_STATIC_IP_ADDR0   192
+#endif
+#ifndef AP_NETWORKING_DEFAULT_STATIC_IP_ADDR1
+    #define AP_NETWORKING_DEFAULT_STATIC_IP_ADDR1   168
+#endif
+#ifndef AP_NETWORKING_DEFAULT_STATIC_IP_ADDR2
+    #define AP_NETWORKING_DEFAULT_STATIC_IP_ADDR2   13
+#endif
+#ifndef AP_NETWORKING_DEFAULT_STATIC_IP_ADDR3
+    #define AP_NETWORKING_DEFAULT_STATIC_IP_ADDR3   14
+#endif
+
+// Default Netmask: 24
+// Note, the netmask is the number of consecutive bits
+#ifndef AP_NETWORKING_DEFAULT_NETMASK
+    #define AP_NETWORKING_DEFAULT_NETMASK       24 // 255.255.255.0
+    // #define AP_NETWORKING_DEFAULT_NETMASK    16 // 255.255.0.0
```

Comment:
Remove? Netmask 24 LGTM.

Reply:
I'd like to keep this in as an example. Maybe I just need to add a comment

Reply:
added more comments